packages/camera/camera_avfoundation/example/ios/RunnerTests/SampleBufferTests.swift

```diff
@@ -232,18 +233,18 @@ final class CameraSampleBufferTests: XCTestCase {
     inputMock.readyForMoreMediaData = true
     sampleAppended = false
     camera.captureOutput(
-      camera.captureVideoOutput, didOutputSampleBuffer: videoSample, from: connectionMock)
+      camera.captureVideoOutput.avOutput, didOutputSampleBuffer: videoSample, from: connectionMock)
     XCTAssertTrue(sampleAppended, "Sample was not appended.")
 
     inputMock.readyForMoreMediaData = false
     sampleAppended = false
     camera.captureOutput(
-      camera.captureVideoOutput, didOutputSampleBuffer: videoSample, from: connectionMock)
+      camera.captureVideoOutput.avOutput, didOutputSampleBuffer: videoSample, from: connectionMock)
     XCTAssertFalse(sampleAppended, "Sample cannot be appended when readyForMoreMediaData is NO.")
   }
 
   func testStopVideoRecordingWithCompletionMustCallCompletion() {
-    let (camera, writerMock, adaptorMock, inputMock, _) = createCamera()
+    let (camera, writerMock, _, _, _) = createCamera()
```

Comment:
we didn't enable treat warning as error?

Reply:
I think we want to enable it for all. But this is not the scope of this PR. Can you file an issue?

Reply:
Done https://github.com/flutter/flutter/issues/165471